lib/consort-report.css

```diff
@@ -62,6 +62,12 @@
   text-decoration: none;
 }
 
```

Comment:
Field validation for radio buttons won't show up unless it's inside the `.form-check` element with the input. I only wanted one feedback message for the whole checkbox group, so I had to override the default styles.